js/src/html/tokenizer.js

```diff
@@ -61,11 +63,13 @@
 
   this.__patterns = {
     word: templatable_reader.until(/[\n\r\t <]/),
+    word_control_flow_close_excluded: templatable_reader.until(/[\n\r\t <}]/),
     single_quote: templatable_reader.until_after(/'/),
     double_quote: templatable_reader.until_after(/"/),
     attribute: templatable_reader.until(/[\n\r\t =>]|\/>/),
     element_name: templatable_reader.until(/[\n\r\t >\/]/),
 
+    angular_control_flow_start: pattern_reader.matching(/\@([a-zA-z]+[\n\t ]*)+[({]/),
```

Comment:
@gergely-gyorgy-both 
the problem looks like you wrote `[a-zA-z]` (lowercase second `z`) and what you wanted was  `[a-zA-Z]` (uppercase second `z`).  Otherwise, ths was fine.